manifests/fedora-coreos-base.yaml

```
@@ -77,13 +77,20 @@ postprocess:
     for x in /etc/yum.repos.d/*modular.repo; do
       sed -i -e 's,enabled=[01],enabled=0,' ${x}
     done
-  # Read the Ignition 2 and Afterburn SSH key fragments, pending better tooling
+  # Disable the default-enabled AuthorizedKeysFile directive so we can
+  # override it in a config fragment
   # https://github.com/coreos/fedora-coreos-tracker/issues/139
+  # https://bugzilla.redhat.com/1824913
   - |
     #!/usr/bin/env bash
     set -xeuo pipefail
     sed -i 's/^AuthorizedKeysFile[[:blank:]]/#&/' /etc/ssh/sshd_config
-    echo -e '\n# Read authorized_keys fragments written by Ignition and Afterburn\nAuthorizedKeysFile .ssh/authorized_keys .ssh/authorized_keys.d/ignition .ssh/authorized_keys.d/afterburn' >> /etc/ssh/sshd_config
+    . /etc/os-release
+    if [ "${VERSION_ID}" = 31 ]; then
```

Minor:

```suggestion
    if [ "${VERSION_ID}" -lt 32 ]; then
```

This isn't relevant *right now*, but will be if e.g. we set up the rawhide stream tracking f33 and this snippet is still here.

---

The test gates the legacy behavior, and we're not going to add any older Fedora versions that require it.  So this should be fine?

---

Yup, you're right. I got my conditions mixed up!